app/assets/stylesheets/typography.scss

```diff
@@ -0,0 +1,49 @@
+.display {
+  font-size: 36px;
+  font-weight: 300;
+  letter-spacing: 0.4;
+  color: $text-primary;
+  font-family: $headline-fonts;
+}
+.headline {
+  font-size: 24px;
+  letter-spacing: 0.3;
+  color: $text-primary;
+  font-family: $headline-fonts;
+}
+.title {
+  font-size: 20px;
+  letter-spacing: 0.2;
+  color: $text-primary;
+  font-family: $headline-fonts;
+}
+.subheader {
```

Comment:
I think doing it like this improves readability and extensibility, and doing it once would only save a few lines.  As it is they're all reading from variables anyway, so if they're being changed those vars will be changed.